data/hai missions.txt

```diff
@@ -435,3 +435,566 @@ mission "Unfettered returning home"
 		dialog
 			`You need not have worried about the reception the Unfettered youth would receive here on Hai-home. Scarcely minutes after you hesitantly contact the Hai government, a happy crowd has gathered around your ship. When he steps out of the hatchway, he looks stunned at being welcomed so warmly. One of the Hai governors thanks you for transporting him, and pays you <payment>.`
 		"reputation: Hai" += 10
+
+
+
+mission "Returning Home"
+	description "Bring Elliot home to his family on <destination>. Elliot ran away from his family as a teenager and hasn't seen them in almost six years."
+	minor
+	source
+		government "Hai"
+	destination
+		attributes "dirt belt"
+		attributes "farming"
+	passengers 1
+	to offer
+		random < 40
+	on offer
+		conversation
+			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
```

Comment:
On reading "You're from the outside . . ."  I imagine Elliot to be 12 - 14 years old, not the 22 that he actually is. Either make the opening sections less child like or make him younger (ran away at 8 would work). Also Victor aged 5 years, might feel better for continuity if he aged 6 year.
Nice story all round, lets hope no one takes years to finish it otherwise the ages at the end will seem really odd.

Reply:
What else would one ask? `"You're from the other side [of the wormhole], aren't you?"`?

Reply:
I think this is fine.
From `FW Pug 2C: Side`:
```
Soon after you land, you spot a man making his way towards your ship, ...

You open your ship's hatch and allow him to come on board. "You came from outside, didn't you?" he says. "Outside of occupied space, I mean." You nod.
```